src/common/modal/index.tsx

```diff
@@ -43,7 +43,7 @@ const StyledDialogActions = styled(DialogActions)`
 `;
 
 const StyledDialog = styled(Dialog)`
-  display: flex;
+  // display: flex;
```

Comment:
removing `display: flex` looks fishy as there are others flex selectors in place

Reply:
Other ones implement their own